GetF!t/SupportingFiles/ColorProvider.swift

```diff
@@ -0,0 +1,18 @@
+import SwiftUI
+
+struct ColorProvider {
+    struct Colors {
+        static let ratingsColor = "ratings"
+    }
+
+    struct BackgroundColor {
+        static let background = "background"
+        static let historyBar = "history-bar"
+        static let gradientTop = "gradient-top"
+    }
+
+    struct ShadowColor {
+        static let dropShadow = "drop-shadow"
+        static let dropHighlight = "drop-highlight"
+    }
+}
```

Comment:
pozri sa [tu](https://github.com/jiledaa/GetFit/blob/review/GetF!t/SupportingFiles/ImageProvider.swift) kde som ti ukázala ako má vyzerať `ColorProvider`

To ako si to spravil ty je stále o dosť lepšie ako si to mal pred tým, ale môj návrh je ešte o trohu lepší, pretože potom môžeš toho písať menej:
```swift
// tvoj kód
Circle()
    .foregroundColor(Color(ColorProvider.BackgroundColor.background))
// môj kód
Circle()
    .foregroundColor(ColorProvider.BackgroundColor.background)
```